lib/module.js

```diff
@@ -2,23 +2,24 @@ const StoryblokClient = require('storyblok-js-client')
 const { pascalCase, normalizeName, isExcluded, addTrailingSlash } = require('./utils')
 const logger = require('./logger')
 
-export default async function storyblokRouterModule(moduleOptions) {
+module.exports = async function storyblokRouterModule (moduleOptions) {
```

Comment:
Why is this better like that?

Reply:
don't mix import style in one file

Reply:
but: https://nuxtjs.org/guide/modules#write-a-basic-module